### ISSUE TITLE
Find Dialogue

### ISSUE THREAD

**Issue by scottlapoint** [microsoft/vscode#159174]
Hello.

I wasn't able to locate this request or functionality (if it exists, so I apologize if it's already there)

I have a vision impairment and need to have things relatively bigger to work.  VS Code has made my life a heck of a lot easier and I wish it existed back in the 90s when I started my career.

That being said there is only one area I was hoping that can be improved.

I find the FIND dialogue box too small and on a big monitor, being up in the upper right sometimes I literally have to stand up to look at it.

I am hoping you can either, allow to make the box MOVEABLE or CENTERED, and an option to make it bigger (aside from increasing the whole work bench size).

I also noticed the new FIND dialogue on the TREE VIEW which is a welcome addition, would it be possible to have it at the TOP of the TREE VIEW ALL THE TIME as an option, there is enough space there that it could exist without having to invoke it. (also an option to make that bigger as well)

Also allow the FILTER selection state to be remembered, I have to turn it on each time I close the FIND box.

Thanks !

**Comment by rebornix** [comment]
@scottlapoint thanks for your detailed feedback, they are very valuable. Currently we only support resizing the find widget in the edtior (drag the vertical bar on the left border of the find widget to resize) but we don't support move the find widget, like what you can do with the debug toolbar. It makes good sense to me.

@joaomoreno can help answer the questions about the Tree/List view find widget. cc our A11y champ @isidorn .

**Comment by isidorn** [comment]
@scottlapoint great feedback, this is super useful! Let me try to answer:

1. Allowing the find widget to be moved horizontally like the debug toolbar makes sense to me
2. I would not allow the find widget to increase size for now. Ideally zooming in the whole workbench helps here, and I would not want that we have specific zooming capability per UI element
3. Preserving filter state in tree find makes sense. @joaomoreno let me know if you would like a new issue for this
4. Preserving tree find across reloads makes sense.  @joaomoreno what do you think? @rebornix why don't we preserve the editor find across reloads?

**Comment by isidorn** [comment]
@scottlapoint yup. Your second suggestion should be covered by my point 4 from above. It would stay visible as long as you do not explicitly close it.
